cloudant-sync-datastore-android/src/main/java/com/cloudant/sync/replication/PeriodicReplicationReceiver.java

```diff
@@ -0,0 +1,46 @@
+package com.cloudant.sync.replication;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.v4.content.WakefulBroadcastReceiver;
+
+/**
+ * This class implements a {@link WakefulBroadcastReceiver} that handles events related to
+ * periodic replication.  It responds to reboot of the device to trigger the resetting of
+ * the {@link android.app.AlarmManager} used to trigger periodic replication, and it
+ * handles the periodic alarms sent by the {@link android.app.AlarmManager} to trigger
+ * the periodic replications.
+ *
```

Comment:
If this is expected to be read in the generated HTML javadoc then it will need `<P></P>` tags or something.